docs/usage.rst

```diff
@@ -308,21 +308,23 @@ the following launch configuration:
 
 .. code:: json
 
-   {
-       "version": "0.2.0",
-       "configurations": [
-           {
-               "name": "Attach to SAM CLI",
-               "type": "node",
-               "request": "attach",
-               "address": "localhost",
-               "port": 5858,
-               "localRoot": "${workspaceRoot}",
-               "remoteRoot": "/var/task",
-               "protocol": "legacy"
-           }
-       ]
-   }
+	{
+		"version": "0.2.0",
+		"configurations": [{
+	        "name": "Attach to SAM CLI",
+	        "type": "node",
+	        "request": "attach",
+	        "address": "localhost",
+	        "port": 5858,
+	        "localRoot": "${workspaceRoot}",

```

Comment:
This should be the ${workspaceRoot}/<path to the function code dir> right? This localRoot only works if you open the directory of the function code and not the whole project.